src/carma-platform/guidance_scripts.py

```diff
@@ -20,9 +21,12 @@
 
 HARDWARE_VEHICLE_STATUS_TOPIC = "/hardware_interface/vehicle_status"
 HARDWARE_VEHICLE_TWIST_TOPIC = "/hardware_interface/vehicle/twist"
-HARDWARE_PACMOD_STEER_REPORT_TOPIC = "/hardware_interface/as/pacmod/parsed_tx/steer_rpt"
-HARDWARE_PACMOD_STEER_CMD_TOPIC = "/hardware_interface/as/pacmod/as_rx/steer_cmd"
```

Comment:
did you remove HARDWARE_PACMOD_STEER_CMD_TOPIC?

Reply:
Yeah I removed that CMD topic because I saw that Report has the command in it too